### PR TITLE
Feature: Style tokens

### LINKED ISSUE #2: POK-2- 🍿 Feature - feat: Add tokens to `style.css`
# Add global styling for colors as tokens to `style.css` 

We can make use of [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) to tokenize our application styling . 

## Colors

Add the following color tokens:

- `--poke-red`
- `--poke-blue`
- `--poke-green`
- `--poke-yellow`
- `--poke-black`
- `--poke-gray`
- `--poke-dark-red`
- `--poke-white`

Please see Figma designs for hex values for each token.

### PR DESCRIPTION
**Pull request by mathiuskitchens**
Add global styling for colors as tokens to style.css

[optional footer(s)]

# Description

Added style tokens for custom poke colors

## Type of change

- [ ] - fix: - patches a bug in the codebase
- [x] - feat: - introduces a new feature
- [ ] - build: - relates to build tooling / project scaffolding
- [ ] - chore: - non-code related changes, like adding packages / libraries
- [ ] - docs: - changes related to documentation

## Related Issue

Does this change have a related issue in the project?

- [x] - Yes - Closes #2 
- [ ] - No

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,6 +5,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fira+Mono:wght@500;700&family=VT323&display=swap"
```
Comment (wijohnst) [inline src/index.html:11]: Did you try using these? Are you sure this worked?

**Review by wijohnst — COMMENTED** [review]
Left a question.

**Comment by wijohnst** [comment]
Closes #3

File: src/index.html (hunk 1 above)
Comment (mathiuskitchens) [reply src/index.html:11]: They are working for me. Is this a reminder to test? Or did they not work when you tried them?

Comment (wijohnst) [reply src/index.html:11]: I didn't try them. Just wanted to make sure that you did!

**Review by wijohnst — COMMENTED** [review]
LGTM.

**Review by wijohnst — APPROVED** [review]

**Pull request merged**